api/golang/core/lib/enclaves/kurtosis_yaml.go

```diff
@@ -12,7 +12,9 @@ const (
 
 // fields are public because it's needed for YAML decoding
 type KurtosisYaml struct {
-	PackageName string `yaml:"name"`
+	PackageName           string            `yaml:"name"`
+	PackageDescription    string            `yaml:"description"`
```

Comment:
are we using this in the PR?

I wonder if we can replace/merge - https://github.com/kurtosis-tech/kurtosis/blob/3b06f221aed3b80030393914a0e1c35310bacf7c/core/server/commons/yaml_parser/kurtosis_yaml.go

lots of dupe code

Reply:
I'll use it in the next PR, and yes I was wondering the same, I'll create a ticket for merging both